assume/common/outputs.py

```diff
@@ -372,13 +373,10 @@ async def on_stop(self):
                 with self.db.begin() as db:
                     df.to_sql("kpis", self.db, if_exists="append", index=None)
         except ProgrammingError as e:
-            self.db.rollback()
+            # self.db.rollback() not working, no rollback function
```

Comment:
remove this line - its not needed anymore , as we have the proper `with self.db.begin()`